libethcore/KeyManager.h

```diff
@@ -33,6 +33,37 @@ namespace eth
 {
 class PasswordUnknown: public Exception {};
 
+enum class PasswordValidation
+{
+	Valid,
+	PasswordIsEmpty,
```

Comment:
You don't have to repeat "Password" everywhere. It is enough to have {Valid, Invalid, Empty, **Too**Short}.